oss-cloud-backend/gitHubApiService.js

```diff
@@ -1,167 +1,160 @@
-const access_token = process.env.GITHUB_ACCESS_TOKEN;
-const Octokit = require("@octokit/rest"); // github api library
+const accessToken = process.env.GITHUB_ACCESS_TOKEN;

```

Comment:
Let's make a `services` directory, and move databaseService & githubApiService there. Now we can make some structure, not to attach everything to root.

Reply:
Services directory created and service files moved.